squid-blockexplorer/src/blocks/utils.ts

```diff
@@ -101,22 +99,24 @@ const SLOT_PROBABILITY = [BigInt(1), BigInt(6)];
  * @return {bigint} - space pledged in bytes
  */
 export function calcSpacePledged(solutionRange: bigint): bigint {
-  const MAX_U64 = BigInt(2 ** 64 - 1);
+  const MAX_U64 = 2n ** 64n - 1n;
+  const PIECE_SIZE = 1048576n;
+  const SLOT_PROBABILITY = [1n, 6n];
 
-  const RECORD_NUM_S_BUCKETS = BigInt(65536);
-  const RECORD_NUM_CHUNKS = BigInt(32768);
-  const SIZE_OF_SOLUTION_RANGE = BigInt(8);
-  const SCALAR_FULL_BYTES = BigInt(32);
+  const RECORD_NUM_S_BUCKETS = 65536n;
+  const RECORD_NUM_CHUNKS = 32768n;
+  const SIZE_OF_SOLUTION_RANGE = 8n;
+  const SCALAR_FULL_BYTES = 32n;
 
-  const history_size =
+  const totalSpacePledged = BigInt(
     MAX_U64 * PIECE_SIZE * SLOT_PROBABILITY[0]
     / RECORD_NUM_S_BUCKETS * RECORD_NUM_CHUNKS
     / solutionRange
     / SLOT_PROBABILITY[1]
     * SCALAR_FULL_BYTES
-    / SIZE_OF_SOLUTION_RANGE;
+    / SIZE_OF_SOLUTION_RANGE);
```

Comment:
Are you sure `BigInt` is necessary here and below? It is just a constructor, but you're operating on bigints (`Xn` is a shortcut for `BitInt(X)` in case number is small enough to not exceed 63-bits or else precision is lost).

I think it should be removed and the rest will remain as is.